benchmark/internal/cireport/upload.go

```diff
@@ -38,22 +33,24 @@ func (rs *fs) WriteFile(dest string, data []byte) (string, error) {
 	// not really useful since we don't have the full path
 	if _, err := os.Stat(dest); err == nil {
 		// file exists
+		logrus.Debugf("file %s exists\n", dest)
 	} else if os.IsNotExist(err) {
 		// file does not exist
+		logrus.Debugf("file %s does not exist\n", dest)
```

Comment:
It seems I'm missing something. Why do we need to check the file it has been successfully written?

Reply:
Good point, this is left out code. Removing.